### PR TITLE
Doc created for the color view in Swift UI

### PR DESCRIPTION
**Pull request by flodega**
### Description

I have created a new entry for Color in swift UI. 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/swiftui/concepts/views/terms/color/color.md
Comment (SSwiniarski) [inline content/swiftui/concepts/views/terms/color/color.md:60]: This needs to link to your image, and your image needs to be moved to the media folder.

File: content/swiftui/concepts/views/terms/color/color.md
Comment (SSwiniarski) [inline content/swiftui/concepts/views/terms/color/color.md:19]:
```suggestion
```

Remove line break.

File: content/swiftui/concepts/views/terms/color/color.md
Comment (SSwiniarski) [inline content/swiftui/concepts/views/terms/color/color.md:53]:
```suggestion
```
Remove line break.

File: content/swiftui/concepts/views/terms/color/color.md
Comment (SSwiniarski) [inline content/swiftui/concepts/views/terms/color/color.md:54]:
```suggestion
```
Remove line break.

File: content/swiftui/concepts/views/terms/color/color.md
Comment (SSwiniarski) [inline content/swiftui/concepts/views/terms/color/color.md:56]:
```suggestion
The code-block above will display an `interface` filled with the color black and the text "Codecademy" in white:
```

File: content/swiftui/concepts/views/terms/color/color.md
Comment (SSwiniarski) [inline content/swiftui/concepts/views/terms/color/color.md:58]: This line is redundant, remove it so the prior line introduces the image.

File: content/swiftui/concepts/views/terms/color/color.md
Comment (SSwiniarski) [inline content/swiftui/concepts/views/terms/color/color.md:34]:
```suggestion
The following example changes the colors to white on black:
```

File: content/swiftui/concepts/views/terms/color/color.md
Comment (SSwiniarski) [inline content/swiftui/concepts/views/terms/color/color.md:51]:
```suggestion
- `Color.black` makes the background color black.
```

File: content/swiftui/concepts/views/terms/color/color.md
Comment (SSwiniarski) [inline content/swiftui/concepts/views/terms/color/color.md:52]:
```suggestion
- `foregroundColor(.white)` sets the text color to white.
```

File: content/swiftui/concepts/views/terms/color/color.md
Comment (SSwiniarski) [inline content/swiftui/concepts/views/terms/color/color.md:17]:
```suggestion
A **Color** view is a way to incorporate colors in the user interface. It can be used to modify the foregrounds and/or backgrounds.
```

Comment (SSwiniarski) [reply content/swiftui/concepts/views/terms/color/color.md:17]: Is `Color` used to change foreground colors? The examples use the `.foregroundColor` view modifier.

File: content/swiftui/concepts/views/terms/color/color.md
Comment (SSwiniarski) [inline content/swiftui/concepts/views/terms/color/color.md:3]:
```suggestion
Description: 'Creates a color view, modifying the background color.'
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@flodega I did a first review and left some comments.

**Comment by SSwiniarski** [comment]
@flodega did you use an account with a separate email? The [CLA ](https://cla-assistant.io/Codecademy/docs?pullRequest=1522) wants that other account to sign.

**Comment by flodega** [comment]
Thanks for your feedback.  I will work on these ASAP to make the changes

**Comment by SSwiniarski** [comment]
@flodega are you still working on this PR? Do you need any assistance?

**Comment by flodega** [comment]
@SSwiniarski yes, I'm still working on this. I'll create a new pull request this week

**Comment by flodega** [comment]
Hello @SSwiniarski , I committed some changes.  Can you see them or do I need to create a new pull request ?  Please let me know.  Thanks

**Comment by SSwiniarski** [comment]
@flodega you don't need to create another PR.

File: content/swiftui/concepts/views/terms/color/color.md
Comment (SSwiniarski) [inline content/swiftui/concepts/views/terms/color/color.md:50]:
```suggestion
- `Color.black` makes the background color black.
```

File: content/swiftui/concepts/views/terms/color/color.md
Comment (SSwiniarski) [inline content/swiftui/concepts/views/terms/color/color.md:51]:
```suggestion
- `foregroundColor(.white)` sets the text color to white.
```

File: content/swiftui/concepts/views/terms/color/color.md
Comment (SSwiniarski) [inline content/swiftui/concepts/views/terms/color/color.md:53]:
```suggestion
The code-block above will display an interface filled with the color black and the text "Codecademy" in white.
```

**Comment by flodega** [comment]
Hello @SSwiniarski, I made the changes and removed the png file but I couldn't find the launch.json file in the folder you mentioned.  Please let me know.

File: content/swiftui/concepts/views/terms/color/color.md
Comment (SSwiniarski) [inline content/swiftui/concepts/views/terms/color/color.md:1]: Remove line break on first line.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@flodega, I had a couple more comments.

**Comment by flodega** [comment]
Hello @SSwiniarski, the changes were made.  Also, do you know how I can resolve the CLA conflict ?  It was caused by the default email from my computer.  

Please let me know

**Comment by SSwiniarski** [comment]
@yangc95 , @KTom101, @caupolicandiaz any idea how to resolve the CLA problem?

**Review by SSwiniarski — APPROVED** [review]
@flodega Good for a second review.

**Comment by KTom101** [comment]
@SSwiniarski @flodega Concerning the CLA conflict, it looks like we had something similar happen [previously](https://github.com/Codecademy/docs/issues/984#issuecomment-1312353504). In the end, the PR was closed out and a new one was created making sure that the correct email and username are configured with Git and GitHub before making the new pull request.

**Comment by flodega** [comment]
@SSwiniarski @KTom101 So would you like me to close the Pull request and create a new one for reviews ?

**Comment by SSwiniarski** [comment]
@flodega that seems to be the solution. But as long as you use the current version of the entry, it will only need one more review.

**Comment by flodega** [comment]
@SSwiniarski Okay, I will close this and push my changes again for a new pull request

**Pull request closed without merging**
